pkg/scheduler/internal/parallelize/parallelism.go

```diff
@@ -23,21 +23,24 @@ import (
 	"k8s.io/client-go/util/workqueue"
 )
 
-const parallelism = 16
+// Parallelism is used the store value from scheduler config
```

Comment:
```suggestion
// Parallelism is amount of parallelism that should be used in scheduler algorithms when scheduling a pod.
```

Reply:
Also, worth adding a quick comment about how this should not be modified after the scheduler has booted up?